.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,13 +1,31 @@
 ---
-name: Bug report
+name: "\U0001F41B Bug report"
 about: Create a bug report to help us improve
+labels: type:bug
 
 ---
 
-**What is the current behavior? Describe the bug**
+## 📝 Provide detailed reproduction steps (if any)
+Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](fiddle.tiny.cloud) or similar.
```

Comment:
for 5 months, anyone who has logged a bug report hasn't been able to click this link because you deleted the `https://` 🤦 

I fixed it on `main`